packages/expansion-hub/src/open-rev-hub.ts

```diff
@@ -96,8 +97,23 @@ async function getSerialPortPathForExHubSerial(
     throw new Error(`Unable to find serial port for ${serialNumber}`);
 }
 
-async function openSerial(serialPortPath: string): Promise<Serial> {
-    let serial = new Serial();
+/**
+ * Closes the given Serial port and removes it from the open serial ports
+ * list. This should be the preferred way to close a Serial port.
+ *
+ * @param serialPort the Serial port to close
+ */
+export function closeSerial(serialPort: SerialPort) {
```

Comment:
Should be `closeSerialPort()` and `openSerialPort()`

Reply:
Fixed